index.css

```diff
@@ -107,3 +111,410 @@ p {
+  padding: 10px;
+}
+
+.line {
+  border-bottom: 1px solid #212121;
+  width: 332px;
+}
+
+.project1 {
+  display: flex;
+  flex-direction: column;
+  padding: 24px;
+}
+
+h3 {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 24px;
+  line-height: 32px;
+  letter-spacing: 0.37px;
+  margin: 0;
+  padding: 4px;
+  color: hsla(218, 76%, 15%, 1);
+}
+
+.prodetail {
+  color: hsla(218, 76%, 15%, 1);
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin: 0;
+}
+
+.blocks {
+  order: 3;
+}
+
+.blocks ul {
+  list-style: none;
+  padding: 0;
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+}
+
+.blocks ul li a {
+  text-decoration: none;
+  border: 1px solid hsla(218, 13%, 59%, 1);
+  padding: 10px;
+  color: hsla(218, 76%, 15%, 1);
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+}
+
+.view {
+  order: 4;
+  display: flex;
+  text-decoration: none;
+  color: white;
+  width: 124px;
+  height: 48px;
+  background-color: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  padding: 3px 12px;
+  justify-content: center;
+  align-items: center;
+}
+
+.pro2 {
+  background: url(probg.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  margin: 10px;
+}
+
+.pro2tit {
+  padding: 180px 0 0 0;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  letter-spacing: 0.37px;
+  color: #fff;
+}
+
+.pro2detail {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: left;
+  padding: 12px 16px 12px 16px;
+}
+
+.pro2bck {
+  order: 5;
+  width: 250px;
+}
+
+.pro2bck ul {
+  list-style: none;
+  padding: 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+.pro2bck ul li a {
+  text-decoration: none;
+  padding: 10px;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.see {
+  order: 6;
+  display: flex;
+  text-decoration: none;
+  color: white;
+  height: 48px;
+  background-color: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  padding: 3px 12px;
+  justify-content: center;
+  align-items: center;
+}
+
+#about {
+  background-color: hsla(22, 6%, 10%, 1);
+  display: grid;
+}
+
+.abtxt {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 10px 24px 50px 0;
+  position: relative;
+}
+
+/*
+.abtxt::after {
+  content: '';
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  width: 132px;
+  height: 318px;
+  background-image: url(abtbg.png);
+  background-repeat: no-repeat;
+} */
```

Comment:
- [ ] [OPTIONAL] I suggest you remove this commenting code in your code base :smiley: